modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java

```diff
@@ -213,6 +196,18 @@ public IgniteToKafkaCdcStreamer(
 
     /** {@inheritDoc} */
     @Override public void start(MetricRegistry mreg) {
+        A.notNull(kafkaProps, "Kafka properties");
+        A.notNull(topic, "Kafka topic");
+
+        kafkaProps.setProperty(KEY_SERIALIZER_CLASS_CONFIG, IntegerSerializer.class.getName());
+        kafkaProps.setProperty(VALUE_SERIALIZER_CLASS_CONFIG, ByteArraySerializer.class.getName());
+
+        cachesIds = cacheNames == null
```

Comment:
We should add check `A.notEmpty(cacheNames)` to ensure caches are set.

Reply:
Done.